app/presenters/publishing_api/historical_accounts_index_presenter.rb

```diff
@@ -24,7 +24,7 @@ def content
         },
         document_type: "historic_appointments",
         public_updated_at: Time.zone.now,
-        rendering_app: Whitehall::RenderingApp::WHITEHALL_FRONTEND,
+        rendering_app: Whitehall::RenderingApp::COLLECTIONS_FRONTEND,
```

Comment:
You'll also need to update the test for this presenter, since the expected hash has been changed.